library/src/main/java/kwasm/format/text/Name.kt

```diff
@@ -15,6 +15,7 @@
 package kwasm.format.text
 
 import kwasm.format.ParseContext
+import kwasm.format.text.token.StringLiteral
```

Comment:
Is this import needed? there are no other changes in the file

Reply:
Yeah, because StringLiteral got moved